style.css

```diff
@@ -254,31 +254,36 @@ input[type="reset"]:disabled,
   outline: none;
 }
 
-.title-bar-controls button[aria-label="Minimize"] {
+.title-bar-controls button[aria-label="Minimize"],
+.title-bar-controls button.minimize {
```

Comment:
Can we ensure an `aria-label` is still present? (i.e. `button[aria-label].minimize`)?

Reply:
Ah didn't think about that! Changed to follow that example, updated documentation as well.